### PR TITLE
Added monitoring for GIT App and GIT API

### PR DESCRIPTION
**Pull request by temitope777**
## Trello
https://trello.com/c/rO6NTNwx/1700-create-alerts-for-all-services

## Context
Added monitoring for GIT App and GIT API

## Changes proposed in this pull request
added GIT App and GIT API to the list of alertable apps in config

## Guidance to review
make test terraform-kubernetes-plan CONFIRM_TEST=yes

### CODE REVIEW

File: cluster/terraform_kubernetes/config/test.tfvars.json [hunk 1]
```diff
@@ -57,15 +57,21 @@
...
       "max_cpu": 0.8
     },
-
     "git-test/get-school-experience-staging": {
        "receiver":"SLACK_WEBHOOK_GSE"
+    },
+    "git-test/get-into-teaching-app-test": {
+       "receiver":"SLACK_WEBHOOK_GIT"
+    } ,
+    "git-test/etintoteachingapi-test": {
```
Comment (saliceti) [inline cluster/terraform_kubernetes/config/test.tfvars.json:66]: Typo

Comment (temitope777) [reply cluster/terraform_kubernetes/config/test.tfvars.json]: corrected

File: cluster/terraform_kubernetes/config/production.tfvars.json [hunk 2]
```diff
@@ -93,5 +95,21 @@
    "tra-production/check-childrens-barred-list-production-worker": {
       "receiver":"SLACK_WEBHOOK_CCBL"
    }
+    },
```
Comment (saliceti) [inline cluster/terraform_kubernetes/config/production.tfvars.json:98]: Extra curly brace?

Comment (temitope777) [reply cluster/terraform_kubernetes/config/production.tfvars.json]: removed

File: cluster/terraform_kubernetes/config/production.tfvars.json [hunk 3]
```diff
@@ -92,6 +92,12 @@
    },
    "tra-production/check-childrens-barred-list-production-worker": {
       "receiver":"SLACK_WEBHOOK_CCBL"
-   }
+   },
+    "git-production/get-into-teaching-app-production": {
```
Comment (saliceti) [inline cluster/terraform_kubernetes/config/production.tfvars.json:96]: The indentation is a bit off. Can you fix the indentation for this and the others in this file?

Comment (temitope777) [reply cluster/terraform_kubernetes/config/production.tfvars.json]: done

**Review by saliceti — APPROVED** [review]

**Comment by temitope777** [comment]
[Create alerts for all services](https://trello.com/c/rO6NTNwx/1700-create-alerts-for-all-services)

**Pull request merged**